static/js/tools/map/place_options.tsx

```diff
@@ -242,6 +242,11 @@ function loadEnclosedPlaces(
           `Sorry, ${place.value.enclosingPlace.name} does not contain places of type ` +
             `${enclosedPlaceType}. Try picking another type or place.`
         );
+        document.getElementById("place-list").innerHTML = "";
+        const select = document.getElementById(
+          "enclosed-place-type"
+        ) as HTMLInputElement;
+        select.value = "";
```

Comment:
instead of directly manipulating the DOM, we should be updating the context which stores information about the options that have been chosen for this tool because we want the context to be in sync with what is displayed. PlaceInfoWrapper has a function called "setEnclosedPlaceType" which allows the enclosedPlaceType to be updated so you should be able to just do `place.setEnclosedPlaceType("")`

Reply:
Got it! I have removed the direct DOM manipulation with `place.setEnclosedPlaceType("")`